include/sentry.h

```diff
@@ -1345,8 +1371,10 @@ SENTRY_EXPERIMENTAL_API sentry_transaction_t *sentry_transaction_start(
  * otherwise.
  *
  * Always takes ownership of `transaction`, regardless of whether the operation
- * was successful or not. If `sentry_set_span` was invoked with `transaction`,
- * this will remove the
+ * was successful or not.
+ *
+ * The Transaction passed in must have an exclusive write lock to be
+ * thread-safe.
```

Comment:
again i'm not really sure about this phrasing together with "takes ownership", to me this kind of conflicts.